bower.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "webfontloader",
+  "main": "webfontloader.js",
+  "version": "1.6.21",
```

Comment:
Is there any chance we could do without the version number? If not, we should change the release script so it auto updates the version on release.